packages/ontario-design-system-complete-styles/gulpfile.js

```diff
@@ -1,16 +1,12 @@
-import { deleteAsync } from 'del';
-
-import autoprefixer from 'gulp-autoprefixer';
+import rimraf from 'rimraf';
```

Comment:
It wasn't imported by our packages. So strange, it was still working, I wonder if it's because it's a peer dependency of several of our dependencies and so it was able to resolve through bundling resolution. I have added it as a dev dependency for both packages that use it in their gulpfile.